definitions/npm/enzyme_v3.x.x/flow_v0.104.x-/enzyme_v3.x.x.js

```diff
@@ -11,63 +11,53 @@ declare module "enzyme" {
   declare type CheerioWrapper = any;
 
   declare class Wrapper<RootComponent> {
-    equals(node: React$Element<any>): boolean,
-    find(selector: UntypedSelector): this,
-    find<T: React$ElementType>(selector: T): ReactWrapper<T>,
-    findWhere(predicate: PredicateFunction<this>): this,
+    at(index: number): this,
+    childAt(index: number): this,
+    children(selector?: UntypedSelector): this,
+    children<T: React$ElementType>(selector: T): ReactWrapper<T>,
+    closest(selector: UntypedSelector): this,
+    closest<T: React$ElementType>(selector: T): ReactWrapper<T>,
+    contains(nodes: React$Element | $ReadOnlyArray<React$Element>): boolean,
```

Comment:
`React$Element` requires a type argument, so wondering why this all pass... 

https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoAlgWwA5wE4AuyYAhgM5gBKApiQMZFR5yZgDketDbqqdcAOzJEC1YQC4qXAgBIAojGqZqAogF4wAHgAm6AG4A+UcM3Bdh3tup0YJTmCgBXAQ3SCwxggApqMSTXpZBSUVAgBKST04dG1UIA

Reply:
I can specify `<any>` explicitly. I think for input arguments it is ok, but for return types - we should make it optionally typeable by the user. So I think I'll need to specify, for example:
`get<T = any>(...): Element<T>`. What do you think?

Reply:
Sounds good. What baffles me is that the tests pass currently without it...

Reply:
Done. I don't know why, looks like a Flow bug to me.